frontend/components/ManageStakingPage/StakingContractSection/index.tsx

```diff
@@ -222,16 +222,10 @@ export const StakingContractSection = ({
   ]);
 
   const contractTagStatus = useMemo(() => {
-    if (activeStakingProgram === stakingProgram)
-      return StakingProgramStatus.Selected;
-
-    // not staked, set the default as the Selected contract
-    if (stakingProgram === defaultStakingProgram)
-      return StakingProgramStatus.Selected;
```

Comment:
need to select the default program if it's not staked
but perhaps not needed, pls check with @oaksprout

Reply:
if required, would suggest:
```typescript
if(activeStakingProgram === null && (stakingProgram === defaultStakingProgram)) 
...
```